### PR TITLE
allow AdminPassword to be modifiable

### PR DESCRIPTION
**Pull request by tzumainn**

### CODE REVIEW

File: lib/egon/undercloud/commands.rb [hunk 1]
```diff
@@ -329,6 +328,9 @@ def self.OSP8_no_registration_no_packages
         echo '  RedisPassword: Rhq8Fd7eEIoPP821Ui' >> openstack-tripleo-heat-templates/overcloud-resource-registry-puppet.yaml
         echo '  HAProxyStatsPassword: veLYjyhxgs7GtQuKXF' >> openstack-tripleo-heat-templates/overcloud-resource-registry-puppet.yaml
 
+        echo 'parameter_defaults:' >> openstack-tripleo-heat-templates/environments/deployment_parameters.yaml
```
Comment (knowncitizen) [inline lib/egon/undercloud/commands.rb:331]: Spacing seems funny here. can you fix it?

Comment (tzumainn) [inline lib/egon/undercloud/commands.rb:331]: It's correct, the output should be something like:

parameter_defaults:
  ComputeCount: '2'

Comment (tzumainn) [inline lib/egon/undercloud/commands.rb:331]: er, except 'ComputeCount' should be indented.  I can't figure out markdown :P

Comment (knowncitizen) [inline lib/egon/undercloud/commands.rb:331]: I follow. thanks.

**Comment by knowncitizen** [comment]
ACK

**Pull request merged**